src/peft/tuners/lora/config.py

```diff
@@ -117,6 +117,26 @@ class LoraConfig(PeftConfig):
             )
         },
     )
+    megatron_config: Optional[dict] = field(
+        default_factory=dict,
+        metadata={
+            "help": (
+                "The TransformerConfig from Megatron, it is used to create LoRA's parallel linear layer. "
+                "You can get it like this, `core_transformer_config_from_args(get_args())`, this two functions are from megatron. "
+                "You need to specify this parameter when you want to loraize the ColumnParallelLinear and RowParallelLinear layers of megatron. "
```

Comment:
Line is too long, trailing whitespace can be removed.

Reply:
> Line is too long, trailing whitespace can be removed.

ok~